src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -127,7 +127,8 @@
   },
   "stsjorbsmod:CULLCard": {
     "NAME": "CULL",
-    "DESCRIPTION": "Permanently decrease this card's damage by !stsjorbsmod:UrMagic! each time it is drawn. NL Deal !D! damage twice. NL If Fatal, permanently increase this card's damage by !M!. NL Exhaust."
+    "DESCRIPTION": "Deal !D! damage twice. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Deal !D! damage twice. NL stsjorbsmod:Wrath is applied twice. NL Exhaust."
```

Comment:
Maybe repeating "if Fatal" from Wrath would make the wording more clear that it's doubling the amount of new wrath stacks gained in the future, rather than doubling the effects of any previously gained stacks? eg, `If Fatal, gain twice the normal stsjorbsmod:Wrath.`?

Reply:
I thought about that, but Wrath is also applied if another non-attack source deals the final damage. So Fatal, or any kind if "if this kills...." is unfortunately incorrect. This is a tricky one to get right without it getting very wordy!

Reply:
Oh, that's a very good point, I had forgotten about that effect of Wrath. Let's try it as-is and iterate if we hear feedback about folks getting confused.